src/Components/Main.jsx

```diff
@@ -44,9 +46,22 @@ export default class Main extends Component {
         url: "/bookmap/books/10805/nightmare-in-shining-armor/",
         title: "Nightmare in Shining Armor",
         author: "Tamar Myers"
+        },
+        {
+        url: "/bookmap/books/10805/nightmare-in-shining-armor/",
+        title: "Overflow Book #1",
+        author: "Overflow Author"
         }
       ]
     }
+
+    this.toggleSidebar = this.toggleSidebar.bind(this);
+  }
+
+  toggleSidebar() {
```

Comment:
If you write this as an arrow function, you don't have to do the .bind(this) step on line 58 because arrow functions inherit their parent's "this" context